etl/steps/data/garden/papers/2023-05-26/ray_et_al_2019.meta.yml

```diff
@@ -0,0 +1,94 @@
+dataset:
+  title: "Climate impact on crop yields (Ray et al. (2019))"
+
+tables:
+  ray_et_al_2019:
+    variables:
+      barley_change_in_yield:
+        title: "Barley yield change"
+        unit: "tonnes per hectare"
+        short_unit: "t/ha"
+      barley_change_with_respect_to_current:
+        title: "Barley yield change with respect to current yield"
+        unit: "%"
+        short_unit: "%"
+      cassava_change_in_yield:
+        title: "Cassava yield change"
+        unit: "tonnes per hectare"
+        short_unit: "t/ha"
+      cassava_change_with_respect_to_current:
+        title: "Cassava yield change with respect to current yield"
+        unit: "%"
+        short_unit: "%"
+      maize_change_in_yield:
+        title: "Maize yield change"
+        unit: "tonnes per hectare"
+        short_unit: "t/ha"
+      maize_change_with_respect_to_current:
+        title: "Maize yield change with respect to current yield"
+        unit: "%"
+        short_unit: "%"
+      oilpalm_change_in_yield:
+        title: "Oilpalm yield change"
```

Comment:
The tiniest quibble, I think it is two words as Oil palm, that is how they refer to it in the main text of the paper.

Reply:
Thanks!